.github/workflows/publish-release-build.yml

```diff
@@ -18,13 +18,13 @@ jobs:
       - uses: ./.github/actions/setup-gradle-build
 
       - name: Build executable and publish to Maven
-        run: ./gradlew clean ktlintCliFiles publishMavenPublicationToMavenCentralRepository --no-daemon --no-parallel --no-configuration-cache
```

Comment:
`--no-daemon` is a no-op, since there are no builds afterwards. The build still executed with a daemon, but it was stopped at the end of the build. Even if, I don't see why the daemon would need to be stopped 🤷 
`--no-parallel`, I assume it was done to prevent issues with publishing. AFAIK there are no issues with the plugin so the workaround is no longer needed
`--no-configuration-cache` - the plugin since the latest release is complatible with configuration cache